charts/astronomer/values.yaml

```diff
@@ -23,7 +23,7 @@ images:
     pullPolicy: IfNotPresent
   houston:
     repository: astronomerinc/ap-houston-api
-    tag: main
+    tag: 0.20.0
```

Comment:
@ryanahamilton @samblackk @adam2k can we update astro-ui in this PR itself? Please change title of PR accordingly.

Reply:
It's updated now.